index.html

```diff
@@ -41,7 +50,196 @@ <h1 class="hero_head">Hey there. I'm Jose Delphin.<br> <span>I'm a software deve
+          <small class="collapse"></small>
+        </div>
+      </li>
+      <li class="nested_flexbox">
+        <div class="items_space">
+
+        </div>
+        <h3>Multi-Post Stories</h3>
+        <h3>Gain + Glory</h3>
+        <ul>
+          <li>Ruby on rails</li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+        <div class="errorButton">
+          <button class="form_button" type="submit">See project</button>
+          <small class="collapse"></small>
+        </div>
+      </li>
+      <li class="nested_flexbox">
+        <div class="items_space">
+
+        </div>
+        <h3>Multi-Post Stories</h3>
+        <h3>Gain + Glory</h3>
+        <ul>
+          <li>Ruby on rails</li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+        <div class="errorButton">
+          <button class="form_button" type="submit">See project</button>
+          <small class="collapse"></small>
+        </div>
+      </li>
+      <li class="nested_flexbox">
+        <div class="items_space">
+
+        </div>
+        <h3>Multi-Post Stories</h3>
+        <h3>Gain + Glory</h3>
+        <ul>
+          <li>Ruby on rails</li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+        <div class="errorButton">
+          <button class="form_button" type="submit">See project</button>
+          <small class="collapse"></small>
+        </div>
+      </li>
+      <li class="nested_flexbox">
+        <div class="items_space">
+
+        </div>
+        <h3>Multi-Post Stories</h3>
+        <h3>Gain + Glory</h3>
+        <ul>
+          <li>Ruby on rails</li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+        <div class="errorButton">
+          <button class="form_button" type="submit">See project</button>
+          <small class="collapse"></small>
+        </div>
+      </li>
+      <li class="nested_flexbox">
+        <div class="items_space">
+
+        </div>
+        <h3>Multi-Post Stories</h3>
+        <h3>Gain + Glory</h3>
+        <ul>
+          <li>Ruby on rails</li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+        <div class="errorButton">
+          <button class="form_button" type="submit">See project</button>
+          <small class="collapse"></small>
+        </div>
+      </li>
+      
+    </ul>
+  </section>
+
+  <header class="background_aboutme">
+    <h2 id="about-me" class="aboutme_head">About me</h2>
+    <section class="grid_container_aboutme">
+      <h3 class="aboutme_text">I can help you build a product ,feature or website. Look through some of my
+        work and experience! If you like what you see and have a project you need coded, don’t hesitate to
+        contact me.</h3>
+      <a class="web_button center_button" href="#">Get My Resume</a>
+    </section>
+  </header>
```

Comment:
Please move this content into the section tag and wrap it with a `div` as opposed to a `header` tag to make your code more semantic and to provide better accessibility on screen readers.


Also, I suggest working on the spacing as well between the contents to match the Figma design at least.

(NOTE: you don't have to copy the exact figures from there as it is just a template so some of the dimensions are not nice when you use it on your actual project. So with your discretion, use a margin that matches the template.)


Your design 👇👇👇


<img width="277" alt="space" src="https://user-images.githubusercontent.com/84930355/205423237-f440912a-301a-45c6-9d5b-882c9fd28beb.png">


Figma 👇👇👇


<img width="223" alt="figgg" src="https://user-images.githubusercontent.com/84930355/205423465-90d5a585-4411-4b53-9b7b-40065b642919.png">

Reply:
done